server/src/utils.ts

```diff
@@ -0,0 +1,73 @@
+import * as lsp from 'vscode-languageserver';
+import {URI} from 'vscode-uri';
+
+enum Scheme {
+  File = 'file',
+}
+;
+
+/**
+ * Extract the file path from the specified `uri`.
+ * @param uri
+ */
+export function uriToFilePath(uri: string): string {
```

Comment:
This method returns wrong value for windows file names. for `file:///e%3A/demo/src/app/app.component.html` returns `/e:/demo/src/app/app.component.html` instead of `e:/demo/src/app/app.component.html`. It's already solved the fix is [here](https://github.com/angular/vscode-ng-language-service/blob/master/server/src/documents.ts#L109-L111)

Reply:
Could you please double check that this is an actual bug? Does tsserver handle the path correctly even with a leading `/`?
I'm sorry I don't have a Windows machine available so I can't test this. In any case, we should file an issue with upstream `vscode-uri` if this is a bug.
https://github.com/microsoft/vscode-uri

Reply:
I have tested in your prototype branch. It doesn't work from windows.
I get this error: 'No config file for /e:/demo/ls-demo/src/app/app.component.ts'.

it starts to work when I change `path` to `fsPath`.
![image](https://user-images.githubusercontent.com/17968027/63621688-3ea66200-c5fd-11e9-98eb-ab78cff9cf71.png)

Reply:
Thank you for the confirmation! I'll fix it.